<ui version="4.0" >
 <class>PlotAsymmetryByLogValueDialog</class>
 <widget class="QDialog" name="PlotAsymmetryByLogValueDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Output workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outWSBox" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Log value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="logBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>First run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstRunBox" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Last run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastRunBox" />
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="browseFirstButton" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseLastButton" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Periods</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="redBox" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="greenBox" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Grouping</string>
   </property>
   <widget class="QWidget" name="layoutWidget_7" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10" >
     <item>
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>Forward spectra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="forwardBox" >
       <property name="mouseTracking" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11" >
     <item>
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>Backward spectra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="backwardBox" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>320</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Time</string>
   </property>
   <widget class="QWidget" name="layoutWidget_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeMinBox" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9" >
     <item>
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timeMaxBox" />
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstRunBox</tabstop>
  <tabstop>browseFirstButton</tabstop>
  <tabstop>lastRunBox</tabstop>
  <tabstop>browseLastButton</tabstop>
  <tabstop>logBox</tabstop>
  <tabstop>outWSBox</tabstop>
  <tabstop>typeBox</tabstop>
  <tabstop>redBox</tabstop>
  <tabstop>greenBox</tabstop>
  <tabstop>forwardBox</tabstop>
  <tabstop>backwardBox</tabstop>
  <tabstop>timeMinBox</tabstop>
  <tabstop>timeMaxBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlotAsymmetryByLogValueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotAsymmetryByLogValueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
